src/main/java/racingcar/Model/MemberState.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.Model;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.StringTokenizer;
+
+public class MemberState {
+    private static final int INIT_DISTANCE = 0;
+
+    public Map<String, Integer> initMemberState(String memberInput) {
+        // HashMap의 순서를 보장하기 위해 LinkedHashMap 사용
+        Map<String, Integer> memberStates = new LinkedHashMap<>();
+        StringTokenizer stringTokenizer = new StringTokenizer(memberInput, ",");
+        while (stringTokenizer.hasMoreTokens()) {
+            String name = stringTokenizer.nextToken();
+            nameException(name);
+            memberStates.put(name, INIT_DISTANCE);
+        }
+
+        return memberStates;
+    }
+
+    public void nameException(String name) {
+        // * 이름 예외 처리
+        // 이름 5자 이하가 아닐 경우
+        if (name.length() > 5) {
+            throw new IllegalArgumentException();
```

Comment:
exception 발생시킬 때 파라미터로 메시지도 같이 넘겨줄 수 있으니까 한 번 공부해보시고 적용해보시면 좋을 것 같아요! 메시지를 넘겨주실거라면 메시지들도 다 상수처리까지 해주면 될 것 같습니다.

```suggestion
            throw new IllegalArguementException("이름은 5자를 넘길 수 없습니다.");
```